ang/civicase-base.ang.php

```diff
@@ -37,11 +37,18 @@
 
 /**
  * Expose settings.
+ *
+ * The default case category is taken from URL first,
+ * or uses `case` as the default.
  */
 function expose_settings(&$options) {
+  $urlCaseCategory = CRM_Utils_Request::retrieveValue('case_type_category', 'String');
+
   $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
   $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
-  $options['defaultCaseCategory'] = strtolower(CRM_Civicase_Helper_CaseCategory::CASE_TYPE_CATEGORY_NAME);
+  $options['defaultCaseCategory'] = $urlCaseCategory
```

Comment:
If this variable will no more store the default case category, I am thinking we should rename to `currentCaseCategory`